### ISSUE TITLE
Unable to run on react typescript template

### ISSUE THREAD

**Issue by franferns** [tomasz-zablocki/redux-orm-typings#4]
Tried running the playground project and it works fine . However when i integrate the new syntax with reactjs typescript template with fields still get the "Maximum stack error". Can you please help me with running the playground code with the attached template react ? I am quite clueless as to what needs to be done to get this to working. 

I have tried with changing tsconfig values with no luck. Thanks in advance. 

[exp-app - Copy.zip](https://github.com/tomasz-zablocki/redux-orm-typings/files/3380941/exp-app.-.Copy.zip)

**Comment by tomasz-zablocki** [comment]
Sure, let me download the attachment and find a convenient approach.

**Comment by tomasz-zablocki** [comment]
Hi @franferns 

First of all, I've reproduced your problem using the sources provided. Since this thing was driving me mad I've inspected it thoroughly. It seems that the approach with properties defined directly on Model classes is not going to work and it has little to do with typings.

Redux-ORM build generates different transpilation outputs for Node.js and loader environments (`lib` and `es` folders in the npm package), which I had wrongly suspected of being responsible for the issue. 

> I have tried with changing tsconfig values with no luck. Thanks in advance.

`create-react-app` pipeline for TypeScript apps is built around Babel, TypeScript source transpilation included and the `tsc` isn't actually used anywhere besides tooling. As a result, there are subtle differences between transpilation outputs and the shape of class properties being one of them - [details in related babel issue](https://github.com/babel/babel/issues/9105). 

Bottom line is that the types are not going to work this way, at least not until the issue is resolved or not with Babel-based TypeScript transpilation. 

I'm back to optimizing types on master branch, In the meantime you might be able to progress by swapping MutableQuerySets with QuerySets where applicable.

**Comment by tomasz-zablocki** [comment]
You can track optimization progress at #5

**Comment by tomasz-zablocki** [comment]
Hi @franferns 

I've managed to get it working 🕺 

I've created a repo with a minimal working example:
https://github.com/tomasz-zablocki/redux-orm-ex-types-react-example

Check the README for details - any questions are welcome.
